toolkits/global/packages/global-skip-link/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@springernature/global-skip-link",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
would this be more of a bug fix (`x.x.X`) than a minor (`x.X.x`) update?

Reply:
happy to change this to bug fix 😄